R_scripts/datahub.R

```diff
@@ -0,0 +1,93 @@
+# datahub.R
+
+# Generate files for PNNL's DataHub repository
+make_datahub <- function() {
+  
+  DIR <- "datahub"
+  
+  if(!dir.exists(DIR)) {
+    message("Creating ", DIR)
+    dir.create(DIR)
+  }
+  
+  MDF <- file.path(DIR, "metadata.txt")
+  if(file.exists(MDF)) {
+    message("Removing old ", MDF)
+    file.remove(MDF)
+  }  
+  
+  cat("Metadata file - PREMIS-ghg data products",
+      date(), sep = "\n", file = MDF)
+  
+  write_df <- function(x) {
```

Comment:
@stephpenn1 I'd suggest taking a hard look at this and following functions, making sure you understand what they do.